### ISSUE TITLE
[onert] Setting OP_BACKEND_* env var produces error at nnpackage_run

### ISSUE THREAD

**Issue by periannath** [Samsung/ONE#2852]
I found this error while debugging #2816

Setting `OP_BACKEND_*` env var always produces error at nnpackage_run

**Example**

```bash
$ OP_BACKEND_Add=cpu BACKENDS=cpu ./Product/armv7l-linux.release/out/bin/nnpackage_run --nnpackage ../nnpkg_tst/inception_v3_2018_04_27
Error during nnfw_session::set_op_backend : _Map_base::at
```

### Debugging
- Error occurs at `OpCode.cc`

https://github.com/Samsung/ONE/blob/6fab873b9f676d3ae18325422bafacbbea3ae6f2/runtime/onert/core/src/ir/OpCode.cc#L36-L44

- str value is `MAX_POOL_2D`
- Map has `MaxPool2D` as key value
   ```cc
     ["MaxPool2D"] = onert::ir::OpCode::MaxPool2D
   ```
- `return map.at(str);` throws an out_of_range exception since it can't find entry for key value
- I think those two strings should match

**Comment by glistening** [comment]
@hseok-oh Could you check do we need this code? As I remember, it was introduced for running a model with non-default layout.

**Comment by hseok-oh** [comment]
`operation_map` in `runtime/onert/api/src/nnfw_api_internal.cc::get_op_backend_string()` is introduced because of mismatch between runtime IR and circle IR.

Below is patch to fix.

```diff
--- a/runtime/onert/api/src/nnfw_api_internal.cc
+++ b/runtime/onert/api/src/nnfw_api_internal.cc
@@ -518,7 +518,7 @@ NNFW_STATUS nnfw_session::register_custom_operation(const std::string &id,
 
 static std::string get_op_backend_string(std::string op)
 {
-#define MAP_MACRO(CircleName, OneRTName) {#CircleName, "OP_BACKEND_" #OneRTName},
+#define MAP_MACRO(CircleName, OneRTName) {#CircleName, #OneRTName},
 
   static std::unordered_map<std::string, std::string> operation_map = {
 #include "OpMap.lst"
@@ -586,7 +586,7 @@ NNFW_STATUS nnfw_session::set_op_backend(const char *op, const char *backend)
     }
 
     auto &opcode_to_backend = _compiler->options().manual_scheduler_options.opcode_to_backend;
-    opcode_to_backend.emplace(onert::ir::toOpCode(op), backend);
+    opcode_to_backend.emplace(onert::ir::toOpCode(key), backend);
   }
   catch (const std::exception &e)
   {
```

**Comment by periannath** [comment]
Fixed by #2861.
